packages-content-model/roosterjs-content-model-core/lib/publicApi/table/normalizeTable.ts

```diff
@@ -6,6 +6,10 @@ import type {
     ContentModelTableCell,
 } from 'roosterjs-content-model-types';
 
+/**
+ * Minimum width for a table cell
+ */
+export const MIN_WIDTH: number = 30;
```

Comment:
Instead of exporting this constant, could we just create a constant in the file that is needed with the same value? (I believe Cell Resizer).
SplitTableCellHorizontally is also doing the same.

Reply:
I think it is better if the value is to reman constant to export it rather than have multiple files with the same value. If the value changes in one place it would have to be manually changed in all places. I think it should also be exported to SplitTableCellHorizontally or maybe removed since the table will be normalised after the split..

Reply:
Also, I have not seen that we are exporting const strings in this packages. I would probably ask Jason when he is back what would be the best practice/guideline for the content model packages

Reply:
I think it is ok to export some const, but agree with Bryan, an exported const should have a more specified name.

For this particular case, instead of export the const directly, can we export a function instead? e.g.

```ts
export function getEffectiveTableCellWidth(originalWidth: number): number {
   return Math.max(originalWidth, MIN_WIDTH);
}
```

So that we don't need to export this width and other places do not need to do the math separately.